ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -3200,3 +3200,34 @@ def test_tensorflow_softsign(
         on_device=on_device,
         features=x[0],
     )
+
+
+# atan
+@handle_frontend_test(
+    fn_tree="tensorflow.math.atan",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
Also tensorflow **atan** accepts also complex dtype could you plz refactor the test dtype to **float_and_complex** instead of **float**